src/antares/service/api_services/study_api.py

```diff
@@ -105,3 +108,66 @@ def delete(self, children: bool) -> None:
             self._wrapper.delete(url)
         except APIError as e:
             raise StudyDeletionError(self.study_id, e.message) from e
+
+    def read_areas(self) -> List:
+        area_list = []
+
+        base_api_url = f"{self._base_url}/studies/{self.study_id}/areas"
+        ui_url = "?ui=true"
+        url_thermal = "clusters/thermal"
+        url_renewable = "clusters/renewable"
+        url_st_storage = "storages"
+        url_properties_form = "properties/form"
+
+        json_resp = self._wrapper.get(base_api_url + ui_url).json()
+
+        for area in json_resp:
+            thermals = dict()  #: t.Dict[any, ThermalCluster]
+            renewables = dict()
+            st_storage = dict()
+
+            area_url = base_api_url + "/" + f"{area}/"
+            json_thermal = self._wrapper.get(area_url + url_thermal).json()
+            json_renewable = self._wrapper.get(area_url + url_renewable).json()
+            json_st_storage = self._wrapper.get(area_url + url_st_storage).json()
+            json_properties = self._wrapper.get(area_url + url_properties_form).json()
+
+            for therm in json_thermal:
+                id_therm = therm.pop("id")
```

Comment:
You could use `thermal` instead of `therm` as i find it clearer. Same goes for `renew`